packages/grafana-runtime/src/config.ts

```diff
@@ -41,6 +41,7 @@ export class GrafanaBootConfig {
   disableSanitizeHtml = false;
   theme: GrafanaTheme;
   pluginsToPreload: string[] = [];
+  enableTransformationsAlpha = false;
```

Comment:
rather than having a special flag every time we want one, what about:
```
featureFlags: string[] = [];
```

Reply:
sgtm